src/seismometer/_api.py

```diff
@@ -23,7 +31,8 @@
 from .data.filter import FilterRule
 from .data.timeseries import create_metric_timeseries
 from .html import template
-from .report.auditing import display_fairness_audit
+from .html.iframe import load_as_iframe
+from .report.auditing import fairness_audit_to_html
 from .report.profiling import ComparisonReportWrapper, SingleReportWrapper
 from .seismogram import Seismogram
 
```

Comment:
This refers to the typehint for function `feature_alerts`: should be `Optional[list[str]] = None`

Don't know why Github doesn't support commenting on collapsed lines.

Reply:
Probably because they weren't modified, but I agree it would be nice if you could.